__docs__/wonder-blocks-accordion/accordion-section.stories.tsx

```diff
@@ -371,6 +414,29 @@ export const CornerKinds: StoryComponentType = {
     },
 };
 
+/**
+ * An AccordionSection can be animated using the `animated` prop.
+ * This animation includes the caret, the expansion/collapse, and the
+ * border radius.
+ *
+ * If the user has `prefers-reduced-motion` opted in, this animation should
+ * be disabled. This can be done by passing `animated={false}` to
+ * the AccordionSection.
+ *
+ * If `animated` is specified both here in the AccordionSection
+ * and within a parent Accordion component, the Accordion's
+ * `animated` value is prioritized.
+ */
+export const WithAnimation: StoryComponentType = {
```

Comment:
**suggestion:** Disable this chromatic snapshot as we cannot visually test this in chromatic.